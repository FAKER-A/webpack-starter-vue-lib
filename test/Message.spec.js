import { shallowMount } from '@vue/test-utils'
import message from '@/message.vue'

describe('message', () => {
  it('renders props.msg when passed', () => {
    const msg = 'new message'
    const wrapper = shallowMount(message, {
      propsData: { msg }
    })
    expect(wrapper.text()).toBe(msg)
    expect(wrapper.html()).toMatchSnapshot()
  })
})