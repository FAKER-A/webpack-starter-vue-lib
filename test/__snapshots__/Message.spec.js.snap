// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`message renders props.msg when passed 1`] = `<h1>new message</h1>`;
